src/main/kotlin/fi/metatavu/lipsanen/projects/milestones/tasks/TaskTranslator.kt

```diff
@@ -23,6 +25,23 @@ class TaskTranslator : AbstractTranslator<TaskEntity, Task>() {
             endDate = entity.endDate,
             milestoneId = entity.milestone.id,
             status = entity.status,
+            assignees = entity.assignees.map { assignee ->
+                TaskAssignee(
+                    id = assignee.id,
+                    taskId = assignee.taskId,
+                    assigneeId = assignee.assigneeId
+                )
+            },
+            userRole = entity.userRole,
+            estimatedDuration = entity.estimatedDuration,
+            estimatedReadiness = entity.estimatedReadiness,
+            attachments = entity.attachments.map { attachment ->
+                TaskAttachment(
+                    id = attachment.id,
+                    taskId = attachment.taskId,
```

Comment:
```suggestion
                    taskId = entity.id,
```
If you choose to use parent-side connection and lose @ManyToOne you can still get task ID from parent entity